src/main/java/com/example/everkan/project/ProjectRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.example.everkan.project;
+
+import com.example.everkan.database.entities.Project;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface ProjectRepository extends JpaRepository<Project, Long> {
+
+    @Query("SELECT project FROM Project project WHERE project.user.id = ?1")
+    Optional<List<Project>> findProjectsByUserId(Long userID);
```

Comment:
Usually, there is no need to make `List` an `Optional`. 
If there are no projects, the `List` can be empty.

Reply:
Changed